tool/tbot/config/config.go

```diff
@@ -214,6 +214,10 @@ func FromCLIConf(cf *CLIConf) (*BotConfig, error) {
 		config.Debug = true
 	}
 
+	if cf.Oneshot {
```

Comment:
I think it would always override in that case, right? If you don't set `cf.Oneshot` we should fall back to the config value (which itself defaults to the zero value)

Reply:
+1 for direct assignment.

Reply:
ah, right